providers/implementations/keymgmt/slh_dsa_kmgmt.c.in

```diff
@@ -373,7 +406,7 @@ static void *slh_dsa_gen(void *genctx, const char *alg)
                                    gctx->entropy, gctx->entropy_len))
         goto err;
 #ifdef FIPS_MODULE
-    if (!slh_dsa_fips140_pairwise_test(key, ctx)) {
+    if (!slh_dsa_fips140_pairwise_test(key, gctx, ctx)) {
         ossl_set_error_state(OSSL_SELF_TEST_TYPE_PCT);
```

Comment:
I'm wondering if the error state should change depending on whether or not the kat_sig parameter is present. If it is present, then you're doing a KAT, so that would be the appropriate error state. If it's not present, you're doing a PCT.

Reply:
Well I am doing both, but it I do not know if it matters in terms of raising the error.